diff --git a/include/configs/mx6sxsabresd.h b/include/configs/mx6sxsabresd.h
index c08e01a99e35..5f918bbf26bc 100644
--- a/include/configs/mx6sxsabresd.h
+++ b/include/configs/mx6sxsabresd.h
@@ -85,7 +85,7 @@
 	"console=ttymxc0\0" \
 	"fdt_high=0xffffffff\0" \
 	"initrd_high=0xffffffff\0" \
-	"fdt_file=undefined\0" \
+	"fdt_file=imx6sx-sdb-btwifi-fmac.dtb\0" \
 	"fdt_addr=0x83000000\0" \
 	"tee_addr=0x84000000\0" \
 	"tee_file=uTee-6sxsdb\0" \
